nightingale_dispatcher/src/nightingale_dispatcher/task.py

```diff
@@ -1,10 +1,17 @@
 #!/usr/bin/env python3
+from enum import Enum
 
 
-class Task:
+class TaskCodes(Enum):
+    # task codes for mission planner
     ERROR = -1
     SUCCESS = 0
     STOCK_ITEMS = 1
     DELIVER_ITEMS = 2
     DISMISS = 3
     WD_TIMEOUT = 9
+
+
+class Task:
```

Comment:
cant we just delete this base task? i dont think its used now

Reply:
If we don't forsee having any common functionality between tasks then we can but otherwise I think it makes sense to be able to define common functionality once